examples/react-native/metro.config.js

```diff
@@ -9,7 +9,12 @@ const INTERNAL_DEPENDENCY_FLAG = '*';
 const INTERNAL_DIRECTORY_ROOTS = ['packages'];
 
 // internal packages used by the RN example
-const INTERNAL_USED_PACKAGES = ['ui', 'react-core', 'react-native'];
+const INTERNAL_DEPENDENCY_DIRECTORY_NAMES = [
```

Comment:
Decided to update the namespace here since the value here and dependency names are not 1:1